replications/internal/queue_management.go

```diff
@@ -126,3 +132,106 @@ func (qm *durableQueueManager) UpdateMaxQueueSize(replicationID platform.ID, max
 
 	return nil
 }
+
+// StartReplicationQueues updates the durableQueueManager.replicationQueues map, fully removing any partially deleted
+// queues (present on disk, but not tracked in sqlite), opening all current queues, and logging info for each.
+func (qm *durableQueueManager) StartReplicationQueues(trackedReplications *influxdb.Replications) error {
```

Comment:
I wouldn't use `influxdb.Replications` here... it gives the impression that the logic has access to / needs more information than is actually being passed in. Could we do a `map[platform.ID]int64` (maybe slightly different key and value types) instead? That should be all the info that's needed